internal/commands/build_test.go

```diff
@@ -426,6 +426,33 @@ func testBuildCommand(t *testing.T, when spec.G, it spec.S) {
 			})
 		})
 
+		when("env vars allow to use quotes in value", func() {
+			it("sets flag variables", func() {
+				mockClient.EXPECT().
+					Build(gomock.Any(), EqBuildOptionsWithEnv(map[string]string{
+						"KEY1": `VALUE"`,
```

Comment:
I'm a bit confused by this - why is it matching 
```
`VALUE"` and not `VALUE""`
```
?

Reply:
`""` escapes `"` - it is the same as in [CSV](https://en.wikipedia.org/wiki/Comma-separated_values) specification. The same escaping is used for other flags: `buildpack` and `tag` since they use `StringSliceVarP`. I just wanted to be consistent with what is already being used.

Sorry for delay - I was on holiday. I see there are some errors about missing tests - I ll try to add them this week.